fmriprep_rodents/workflows/base.py

```diff
@@ -147,7 +147,7 @@ def init_single_subject_wf(subject_id):
             f"task <{task_id or 'all'}>. All workflows require BOLD images."
         )
 
-    workflow = Workflow(name=f"single_subject_{subject_id}_wf")
+    workflow = Workflow(name=name)
```

Comment:
You'll need to revert this one
```suggestion
    workflow = Workflow(name=f"single_subject_{subject_id}_wf")
```

Reply:
Oh - in a versionI was looking at, `name` was defined on line 132. Happy to oblige 👍🏼